src/routers/account.router.js

```diff
@@ -0,0 +1,13 @@
+import express from "express";
+import { AccountController } from "../controllers/account.controller.js";
+import { authenticateToken } from "../middlewares/auth.middleware.js";
+
+export const accountRouter = express.Router();
+
+accountRouter.use(authenticateToken);
+
+accountRouter.get("/accounts/:user_id", AccountController.findById);
+accountRouter.get("/accounts", AccountController.findAllAccounts);
+accountRouter.put("/accounts", AccountController.updateAccount);
+accountRouter.delete("/accounts", AccountController.deleteAllAccounts);
+accountRouter.delete("/accounts/:user_id", AccountController.deleteOneAccount);
```

Comment:
in previus comment we add router path /accounts

```suggestion
accountRouter.get("/:user_id", AccountController.findById);
accountRouter.get("/", AccountController.findAllAccounts);
accountRouter.put("/", AccountController.updateAccount);
accountRouter.delete("/", AccountController.deleteAllAccounts);
accountRouter.delete("/:user_id", AccountController.deleteOneAccount);
```